ci/validate.sh

```diff
@@ -26,26 +26,31 @@ for target in image_builder/gce.json base_images/cloud.json \
   fi
 done
 
+### The following checks only apply if validating a PR
+if [[ -z "$CIRRUS_PR" ]]; then
+  echo "Not validating IMG_SFX changes outside of a PR"
```

Comment:
Should this have an `exit`?

Reply:
Lol woops!  I swore it did at one point, must have clobbered it on accident.  Thanks for noticing.